packages/core/src/reporters/clear-text-reporter.ts

```diff
@@ -109,8 +109,13 @@ export class ClearTextReporter implements Reporter {
     this.writeLine(`Ran ${(totalTests / systemUnderTestMetrics.metrics.totalMutants).toFixed(2)} tests per mutant on average.`);
   }
 
+  private getStatusStringForMutant(mutant: MutantModel): string {
```

Comment:
Suggestion: `statusLabel`. The fact that it is for a mutant is communicated by the parameter type 🤷‍♀️